Client/core/CCore.cpp

```diff
@@ -60,13 +60,13 @@ static HMODULE WINAPI SkipDirectPlay_LoadLibraryA(LPCSTR fileName)
         const fs::path inLaunchDir = fs::path{FromUTF8(GetLaunchPath())} / "enbseries" / "enbhelper.dll";
 
         if (fs::is_regular_file(inLaunchDir, ec))
-            return Win32LoadLibraryA(inLaunchDir.u8string().c_str());
+            return Win32LoadLibraryA(inLaunchDir.string().c_str());
 
         // Try to load enbhelper.dll from the GTA install directory second.
         const fs::path inGTADir = g_gtaDirectory / "enbseries" / "enbhelper.dll";
 
         if (fs::is_regular_file(inGTADir, ec))
-            return Win32LoadLibraryA(inGTADir.u8string().c_str());
+            return Win32LoadLibraryA(inGTADir.string().c_str());
```

Comment:
leave it as u8string for utf8 compatibility

Reply:
c_str in 'u8' returns char8_t, which is not compatible with LPCSTR

Reply:
Right, it changes in the C++20. Nevermind then

Reply:
Can I close this and the like?

Reply:
Sure

Reply:
See [comment](https://github.com/multitheftauto/mtasa-blue/pull/3636#discussion_r1729407241)